test/unit/Query/QueryCollection.test.php

```diff
@@ -4,24 +4,71 @@
 use Gt\Database\Connection\DefaultSettings;
 use Gt\Database\Connection\Driver;
 use Gt\Database\Result\ResultSet;
+use PHPUnit_Framework_MockObject_MockObject;
 
 class QueryCollectionTest extends \PHPUnit_Framework_TestCase {
+/** @var  QueryCollection */
+private $SUT;
```

Comment:
What does this SUT mean? Would it be more obvious to call it `$this->queryCollection`?